src/pages/App.jsx

```diff
@@ -17,6 +17,7 @@ export const App = () => {
           <Route path="/" element={<Landing />} />
           <Route path="/about" element={<About />} />
           <Route path="/categories" element={<Categories />} />
+          <Route path="/walletviewpage" element={<WalletViewPage />} />
```

Comment:
wallet view should have an id in url since you can view more than one wallet and we need a way to differentiate